homeassistant/components/camera/synology.py

```diff
@@ -0,0 +1,104 @@
+"""
+Support for IP Cameras.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.generic/
+"""
+import logging
+
+import json
+from json import loads
+

```

Comment:
Usually it's easier to spot if the module name is present when using very common function. So on line 46 would it be `json.loads`.